requirements.txt

```diff
@@ -1,14 +1,25 @@
+# Used in local execution and dev.
 ipython
 requests
+
+# Used by all local and Lambda-based code.
 strict-rfc3339
+
+# To support a11y scanner:
 pyyaml
 
-# to support sslyze scanner
+# Ubuntu 14.04 has an old version of six
+six>=1.6
+
+# To support pshtt scanner (and to stay fresh):
+git+git://github.com/dhs-ncats/pshtt.git#egg=pshtt
```

Comment:
This pulls `pshtt` from GitHub, at least for now, since PyPi has been lagging behind.

Also, this particular build is making use of an unmerged PR to `pshtt`, in https://github.com/dhs-ncats/pshtt/pull/136, which means that any local execution needs to have `pshtt` installed from that branch, and any Lambda execution needs to have had its deps compiled with `pshtt` on that branch (which the .zip in this PR has been).

Once that PR is merged, and ideally when PyPi has been updated, this can all be updated.